eng/common/testproxy/test-proxy-docker.yml

```diff
@@ -6,6 +6,28 @@ steps:
         $(Build.SourcesDirectory)/eng/common/scripts/trust-proxy-certificate.ps1
     displayName: 'Language Specific Certificate Trust'
 
+  # windows has a startup bug where the service is unavailable. See Azure/azure-sdk-tools#2122
+  - pwsh: |
```

Comment:
We think that `docker info` is kicking the tires here.

Reply:
I'm going to swap within `docker-start-proxy.ps1` to just do a retry on the startup. Probably not going to go with this.